concepts/conditionals-if/about.md

```diff
@@ -40,7 +43,24 @@ function getVal(connected bool) int {
 }
 ```
 
-Coming from other languages one may be tempted to try to use one-line conditionals. Go does not support ternary operators or one-line conditionals. This is a purposeful design decision to limit the amount of possibilities, making code simpler and easier to read.
+If statements can also use a short initialization statement that can be used to initialize one or more variables for the if statement.
+For example:
+
+```go
+num := 7
+if v := 2 * num; v > 10 {
+    fmt.Println(v)
+} else {
+    fmt.Println(num)
+}
+// Output: 7
```

Comment:
Usually these are always used with functions so I am contemplating whether to make this look a bit more like it would look in real Go code even if the example is a little more abstract then, e.g. like this:
```suggestion
num := 8
if ok := isEven(num); ok {
    // ...
}
```

I also removed the else part and the code inside as they make it harder to see what the example is actually about.

Reply:
So, the problem I had with going this route is that it would beg the question of why use the initialization at all? For example, that one could be re-written as:

```go
if isEven(num) {
  // ...
}
```

I contrived the example that I wrote specifically to avoid that problem (since you want to print out the value returned). I suppose I could have the example I wrote invoke a function called `double`, but that means you have to infer what that function does in order to understand which branch it chose. In the end, I think removing the need for a function at all is simpler for the student.